traitsui/examples/demo/Extras/Image_editor_demo.py

```diff
@@ -15,20 +15,18 @@
 from traits.api \
     import HasTraits, Str
 
-from traitsui.api \
-    import View, VGroup, Item
+import traitsui.api
 
 from traitsui.api \
-    import ImageEditor
+    import View, VGroup, Item, ImageEditor
 
 from pyface.image_resource \
     import ImageResource
 
 # Constants:
 
 # Necessary because of the dynamic way in which the demos are loaded:
-search_path = [join(dirname(traits.api.__file__),
-                    '..', '..', 'examples', 'demo', 'Extras')]
+search_path = [join(dirname(__file__))]
```

Comment:
The `join` is actually redundant now that there are no fragments to join. Sorry about that. `[dirname(__file__)]` would do.